cylc/flow/job_runner_handlers/slurm.py

```diff
@@ -40,8 +40,9 @@
 
 .. note::
 
-   Since not all SLURM commands have a short form, cylc requires
-   the long form directives.
+   * Cylc requires long form directives (e.g. ``--begin`` not ``-b``).
```

Comment:
Is this actually true?

Wouldn't `-b =` work?

Reply:
No, because Slurm won't accept that:

```console

> cat > hello.sh <<__HERE__
+ #!/bin/bash
+ #SBATCH -b=60
+ echo "Hello World"
+ __HERE__
> chmod +x hello.sh 
> sbatch hello.sh 
sbatch: invalid option -- '='
sbatch: invalid option -- '6'
sbatch: invalid option -- '0'
Submitted batch job 3309508
```

`-H =` does work, however.

Reply:
Note, I didn't give an argument with the `-b` option above, just left it empty which should be valid?

E.G:

```
[directrives]
-H =
```
Should result in:
```
#SBATCH -H
```

Which I would expect to work?

Reply:
So Cylc slurm support doesn't support short options which require arguments?

Reply:
> So Cylc slurm support doesn't support short options which require arguments? Because slurm doesn't support the `=` operator with short options?

This

Reply:
> Note, I didn't give an argument with the `-b` option above, just left it empty which should be valid?

Yes, `-b` is, much to my surprise, valid. Pointless, but valid.